backend/app/services/shared_task.py

```
@@ -552,15 +552,11 @@ def get_public_shared_task(
                 status_code=404,
-                detail="This shared task is no longer available. It may have been deleted by the owner."
+                detail="This shared task is no longer available. It may have been deleted by the owner.",
             )
 
         # Get user info for sharer name
-        user = (
-            db.query(User)
-            .filter(User.id == user_id, User.is_active == True)
-            .first()
-        )
+        user = db.query(User).filter(User.id == user_id, User.is_active == True).first()
```

Ruff (0.14.7)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)